app/auth-portal/src/pages/WorkspacesPage.vue

```diff
@@ -240,4 +245,22 @@ const getSubscriptionTier = computed(() => {
 
   return "FREE SUBSCRIPTION";
 });
+
+const searchString = ref("");
+const onSearch = (search: string) => {
+  searchString.value = search.trim().toLocaleLowerCase();
+};
+
+const filteredWorkspaces = computed(() => {
+  return sortedWorkspaces(filterWorkspacesBySearchString(workspaces.value));
+});
+
+const filterWorkspacesBySearchString = (workspaces: Workspace[]) => {
+  return _.filter(workspaces, (w) => {
+    if (w.displayName.toLowerCase().includes(searchString.value)) {
+      return true;
+    }
+    return false;
+  });
```

Comment:
Clever! `includes` relying on the `""` default value will work too, as shown in the demo.